rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -13,139 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
I would like to see a note in this file explaining where the new defaults are defined. It might be a little confusing for someone to go into ``/etc/openstack_deploy/user_rpco_variables_defaults.yml`` and not find anything.